lib/jekyll.rb

```diff
@@ -32,6 +32,9 @@ def require_all(path)
 require "colorator"
 
 SafeYAML::OPTIONS[:suppress_warnings] = true
+ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
```

Comment:
I'll stick with what I chose as I pleases me.

On Sat, Sep 10, 2016, 3:48 AM Saleem Rashid <notifications@github.com>
wrote:

> In lib/jekyll.rb
> <https://github.com/jekyll/jekyll/pull/5344#discussion_r78274115>:
>
> > @@ -32,6 +32,9 @@ def require_all(path)
> >  require "colorator"
> >
> >  SafeYAML::OPTIONS[:suppress_warnings] = true
> > +ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
>
> Really? 😏
>
> Try Parallel.processor_count
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5344/files/f8e64e20e47f8bbb29c7b202747a1875a058ada8#r78274115>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGFs8xmIJyqc5MhnEZ6fHVtSZL40ajmks5qom7KgaJpZM4J5fpq>
> .
>